tests/util.go

```diff
@@ -32,11 +32,15 @@ type Result struct {
 var KubeVirtOcPath = ""
 
 const (
-	CDI_LABEL_KEY        = "app"
-	CDI_LABEL_VALUE      = "containerized-data-importer"
-	CDI_LABEL_SELECTOR   = CDI_LABEL_KEY + "=" + CDI_LABEL_VALUE
+	TemplateNS           = "openshift"
```

Comment:
why you need to create another namespace, is there anything you cant do in `kubevirt-test-default` ?

Reply:
The vm templates are residing in ns 'openshift', we need to test it to ensure they're working.

Reply:
It

> this PR is similar to my #354, we can figure it out how to combine those two

There are would be some tests are similar at the setup, it would be okay.
We should not combine different tests for different purposes.